src/Fmt_ast.ml

```diff
@@ -3133,6 +3201,7 @@ and fmt_module_expr c ({ast= m} as xmod) =
   let ctx = Mod m in
   let {pmod_desc; pmod_loc; pmod_attributes} = m in
   let c = update_config c pmod_attributes in
+  (*maybe_disabled c pmod_loc @@ fun () -> *)
```

Comment:
?

Reply:
This should return a `block`. I did have time to look at this.

Reply:
Ah yes, ugh, ok.